### ISSUE TITLE
optimize same time as plot

### ISSUE THREAD

**Issue by jxer** [Mirkic7/mdcct#7]
can we direct optimize? or redirect output your requiring double the hd space on the same drive just to optimize since we can't write to another hd temporarily?

**Comment by BobbyT** [comment]
@jxer I think this could be it?

> Recently optimised mjminer plotter to produce fully optimised plots directly.

Source: https://forums.burst-team.us/topic/5998/mjminer-plotter-fork-for-optimized-plots-on-macos-cpu-avx2-sse4

Detailed Description:
https://steemit.com/burst/@k06a/efficient-burst-mining-on-macos-linux

**Comment by jxer** [comment]
@BobbyT 

yep got it working, had to manually compile, but i have it plotting right now, seems decent
